inbound_test.go

```diff
@@ -101,3 +101,44 @@ func TestInboundConnection(t *testing.T) {
 		require.NoError(t, err, "Call failed")
 	})
 }
+
+func TestInboundConnection_CallOptions(t *testing.T) {
+	ctx, cancel := NewContext(time.Second)
+	defer cancel()
+
+	WithVerifiedServer(t, nil, func(serverCh *Channel, serverHostPort string) {
```

Comment:
can you make this test use `testutils.WithTestServer`, and then instead of `testutils.NewServerChannel` we should use `ts.NewServer` and `ts.NewClient` for the client.

there's extra verification for things created using the `testutils.TestServer`